metriq_gym/dispatch_clops.py

```diff
@@ -18,36 +18,35 @@
 def main():
     args = parse_arguments()
 
-    if not args.token:
-        raise ValueError("CLOPS benchmark requires an IBMQ token!")
-
-    QiskitRuntimeService.save_account(channel="ibm_quantum", token=args.token, set_as_default=True, overwrite=True)
+    if args.token:
+        QiskitRuntimeService.save_account(channel="ibm_quantum", token=args.token, set_as_default=True, overwrite=True)
 
     logging.info(f"Dispatching CLOPS job with n={args.num_qubits}, shots={args.shots}, trials={args.trials}, backend={args.backend}, confidence_level={args.confidence_level}, jobs_file={args.jobs_file}")
 
     clops = clops_benchmark(
-        QiskitRuntimeService(),
-        args.backend,
-        width = args.num_qubits,
-        layers = args.num_qubits,
-        shots = args.shots
+        service=QiskitRuntimeService(),
+        backend_name=args.backend,
```

Comment:
Should we call this `backend_name` or `backend`? The `backend` object has an attribute of `.name`, so I wonder if this is more appropriately called `backend` as a function argument. 

This would be a suggested change for the `clops_benchmark` function (not for what you are providing to it).

Reply:
Same reply with regards to all three comments: the class `clops_benchmark` comes from [`clops_benchmark](https://github.com/unitaryfund/metriq-gym/blob/cb8c1d5ed727a9d91f78b9c29eb4834c6a6703d0/metriq_gym/hardware/clops_benchmark.py), which in truns is a copy of https://github.com/qiskit-community/qiskit-device-benchmarking/blob/10dc67e7ecad15b02f7cf95993c07eb876c97489/qiskit_device_benchmarking/clops/clops_benchmark.py. I would rather leave it as is for now, so it will be easy to sync it with "upstream".